solo-chains/runtime/starlight/src/genesis_config_presets.rs

```diff
@@ -253,10 +258,45 @@ fn starlight_testnet_genesis(
         .collect();
 
     let para_ids: Vec<_> = container_chains
-        .into_iter()
+        .iter()
+        .cloned()
         .map(|(para_id, genesis_data, _boot_nodes)| (para_id, genesis_data, None))
         .collect();
 
+    // In order to register container-chains from genesis, we need to register their
+    // head on the relay registrar. However there is no easy way to do that unless we touch all the code
+    // so we generate a dummy head state for it. This can be then overriden (as zombienet does) and everything would work
+    // TODO: make this cleaner
+    let registrar_para_ids: Vec<_> = container_chains
```

Comment:
As this variable holds not only the paraIds but also their info, a more representative name could be `registrar_para_ids_info`.

Reply:
fixed